backend/core/validations/user_validation/UserPersonalDataValidation.js

```diff
@@ -0,0 +1,11 @@
+const namesValidation = require('./NameValidation.js');
```

Comment:
I just looked into it and it's fine to use snake case for folder names, but then the js files should also be lower case.
Check out https://stackoverflow.com/questions/18927298/node-js-project-naming-conventions-for-files-folders/20885809 for reference.